apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/model/entity/OrganizationEntityTest.java

```diff
@@ -0,0 +1,163 @@
+package org.sagebionetworks.openchallenges.organization.service.model.entity;
+
+import java.time.OffsetDateTime;
+import java.util.Collections;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class OrganizationEntityTest {
+
+  private static Long id;
+  private static String name;
+  private static String email;
+  private static String login;
+  private static String avatarKey;
+  private static String websiteUrl;
+  private static Integer challengeCount;
+  private static String description;
+  private static OffsetDateTime createdAt;
+  private static OffsetDateTime updatedAt;
+  private static String acronym;
+  private static OrganizationEntity entityFromConstructor;
+  private static OrganizationEntity entityFromConstructor2;
+
+  @BeforeEach
+  public void setup() {
+    id = 1L;
+    name = "Test Organization";
+    email = "test@example.com";
+    login = "testorg";
+    avatarKey = "avatarKey";
+    websiteUrl = "https://example.com";
+    challengeCount = 5;
+    description = "Test description";
+    createdAt = OffsetDateTime.now();
+    updatedAt = OffsetDateTime.now();
+    acronym = "TO";
+
+    entityFromConstructor = new OrganizationEntity();
+
+    entityFromConstructor.setId(id);
+    entityFromConstructor.setName(name);
+    entityFromConstructor.setEmail(email);
+    entityFromConstructor.setLogin(login);
+    entityFromConstructor.setDescription(description);
+    entityFromConstructor.setAvatarKey(avatarKey);
+    entityFromConstructor.setWebsiteUrl(websiteUrl);
+    entityFromConstructor.setChallengeCount(challengeCount);
+    entityFromConstructor.setCreatedAt(createdAt);
+    entityFromConstructor.setUpdatedAt(updatedAt);
+    entityFromConstructor.setAcronym(acronym);
+    entityFromConstructor.setCategories(Collections.emptyList());
+    entityFromConstructor.setChallengeContributions(Collections.emptyList());
+
+    entityFromConstructor2 = new OrganizationEntity();
+
+    entityFromConstructor2.setId(id);
+    entityFromConstructor2.setName(name);
+    entityFromConstructor2.setEmail(email);
+    entityFromConstructor2.setLogin(login);
+    entityFromConstructor2.setDescription(description);
+    entityFromConstructor2.setAvatarKey(avatarKey);
+    entityFromConstructor2.setWebsiteUrl(websiteUrl);
+    entityFromConstructor2.setChallengeCount(challengeCount);
+    entityFromConstructor2.setCreatedAt(createdAt);
+    entityFromConstructor2.setUpdatedAt(updatedAt);
+    entityFromConstructor2.setAcronym(acronym);
+    entityFromConstructor2.setCategories(Collections.emptyList());
+    entityFromConstructor2.setChallengeContributions(Collections.emptyList());
+  }
+
+  @Test
+  public void
+      OrganizationEntityBuilderArguments_ShouldReturnArgumentsPassed_WhenOrganizationEntityBuilt() {
```

Comment:
Review title: the "BuilderArguments" don't "return" anything. You could say that the org entity returns the arguments passed when built with the class builder.

Reply:
Changed the name to `OrganizationEntity_ShouldReturnArgumentsPassed_WhenOrganizationEntityBuiltWithClassBuilder`